services/resource-remover.js

```diff
@@ -1,9 +1,21 @@
 'use strict';
+var Interface = require('forest-express');
+var CompositeKeysManager = require('./composite-keys-manager');
 
 function ResourceRemover(model, params) {
+  var schema = Interface.Schemas.schemas[model.name];
+
   this.perform = function () {
+    var where = {};
+    var compositeKeysManager = new CompositeKeysManager(model, schema, params);
+
+    if (schema.isCompositePrimary) {
+      where = compositeKeysManager.getRecordConditions(params.recordId);
+    } else {
+      where[schema.idField] = params.recordId;
+    }
```

Comment:
We need to factorise this code (duplicated in at least 3 places: resource-getter, resource-updater, resource-remover)